packages/x-components/build/docgen/utils.js

```diff
@@ -50,7 +50,7 @@ function getDocumentFileDestination(file, config) {
 function generateDestination(folder, regex, file) {
   const match = regex.exec(file);
   if (match) {
-    const [, path, componentName] = match;
+    const { path, componentName } = match.groups;
```

Comment:
Even though this change works perfectly I realised that if the component is the root of the `components` directory, the path is not matched:

![image](https://user-images.githubusercontent.com/3502075/137754626-7945f543-da5e-41fd-aee7-96ad7b91a365.png)

Apparently it should fails as the method should returns:
```
return `${folder}//x-components.${componentName}.md`;
```
But it works.